Dockerfile

```diff
@@ -36,6 +27,15 @@ RUN git clone https://www.github.com/singularityware/singularity.git
 WORKDIR singularity
 RUN ./autogen.sh && ./configure --prefix=/usr/local && make && make install
 
+RUN mkdir /code
+ADD . /code
+RUN /opt/conda/bin/pip install setuptools
+RUN /opt/conda/bin/pip install scif
+
+RUN scif install /code/sregistry-cli.scif
+ENTRYPOINT ["sregistry"]
+
 WORKDIR /code
+RUN rm /usr/bin/python && ln -s /opt/conda/bin/python /usr/bin/python
```

Comment:
```
Step 16/17 : RUN rm /usr/bin/python && ln -s /opt/conda/bin/python /usr/bin/python
 ---> Running in ee1a2bf31b09
rm: cannot remove '/usr/bin/python': No such file or directory
```

Perhaps just `ln -sf /opt/conda/bin/python /usr/bin/python`?

Or `test -f /usr/bin/python || ln -s /opt/conda/bin/python /usr/bin/python`?

Reply:
What's the difference between those two?

Reply:
Okay thanks! My preference is for the first, since we want to ensure that the link is correct.